### PR TITLE
fix: fix types for `cloudflare:queue` args

### PR DESCRIPTION
**Pull request by dalbodeule**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The queue event(MessageBatch) is wrong types.
What was originally supposed to be MessageBatch<unknown>, I looked it up today and found that it was written as CloudflareEmailContext.
I just fixed it.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

https://github.com/unjs/nitro/issues/2587#issuecomment-2240766921

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
   "name": "nitro-devcontainer",
   "forwardPorts": [3000],
-  "image": "node:20.14.0",
+  "image": "node:20.15.1",
```
Comment (pi0) [inline .devcontainer/devcontainer.json:6]:
```suggestion
  "image": "node:20.14.0",
```

**Review by pi0 — APPROVED** [review]
Thanks

**Comment by dalbodeule** [comment]
additional fix: types for cloudflare:queue args

CloudflareEmailContext.raw to ReadableStream<Uint8Array> from ReadableStream

**Comment by dalbodeule** [comment]
I have something additional to ask.

Testing with Nuxt nightly. The error occurs during the build process.

```
 ERROR  Cannot resolve "cloudflare:email" from "/Users/jioo0224/Desktop/nodejs/sh0rt-kr-email-worker/node_modules/cloudflare-email-kit/dist/index.js" and externals are not allowed!                                                 10:49:18 AM

  at Object.resolveId (node_modules/nitro/dist/rollup/index.mjs:1570:17)
  at async PluginDriver.hookFirstAndGetPlugin (node_modules/rollup/dist/es/shared/node-entry.js:19708:28)
  at async resolveId (node_modules/rollup/dist/es/shared/node-entry.js:18389:26)
  at async ModuleLoader.resolveId (node_modules/rollup/dist/es/shared/node-entry.js:18792:15)
  at async Object.resolveId (node_modules/@rollup/plugin-commonjs/dist/es/index.js:590:10)
  at async PluginDriver.hookFirstAndGetPlugin (node_modules/rollup/dist/es/shared/node-entry.js:19708:28)
  at async resolveId (node_modules/rollup/dist/es/shared/node-entry.js:18389:26)
  at async ModuleLoader.resolveId (node_modules/rollup/dist/es/shared/node-entry.js:18792:15)
  at async ModuleLoader.resolveDynamicImport (node_modules/rollup/dist/es/shared/node-entry.js:19196:82)
  at async node_modules/rollup/dist/es/shared/node-entry.js:19083:32
```

I would like to ask whether this is also a problem with Nitro and, if so, where I should fix it.

**Comment by dalbodeule** [comment]
I don't know think so.

Maybe [this rule](https://github.com/unjs/nitro/blob/79c58b919828fd108a3887fd8bd6d37d2928736f/src/rollup/config.ts#L406) or "cloudflare-email-kit" plugin is fault.

Before, "cloudflare-email-kit" module's dependency is all of "browser" dependencies. So, I think that rules is problem...

**Comment by pi0** [comment]
Thanks for PR. Fixed via https://github.com/unjs/nitro/pull/2711

**Pull request closed without merging**